### PR TITLE
[BFCL] Add DeepSeek-V2.5, DeepSeek-Coder-V2-Instruct-0724, DeepSeek-Coder-V2-Lite-Instruct, DeepSeek-V2-Chat-0628, DeepSeek-V2-Lite-Chat

### PR DESCRIPTION
**Pull request by moonlight1431**
Add the following new models to the leaderboard:
  - `deepseek-ai/DeepSeek-V2.5`
  - `deepseek-ai/DeepSeek-Coder-V2-Instruct-0724`
  - `deepseek-ai/DeepSeek-Coder-V2-Lite-Instruct`
  - `deepseek-ai/DeepSeek-V2-Chat-0628`
  - `deepseek-ai/DeepSeek-V2-Lite-Chat`

### CODE REVIEW

**Comment by HuanzhiMao** [comment]
Could you update the [deepseekHandler](https://github.com/ShishirPatil/gorilla/blob/main/berkeley-function-call-leaderboard/bfcl/model_handler/oss_model/deepseek.py) with DeepSeek-Coder-V2 as well? https://huggingface.co/deepseek-ai/DeepSeek-Coder-V2-Instruct

File: berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py [hunk 1]
```diff
@@ -665,6 +695,11 @@
     "command-r-plus-FC-optimized": 3,
     "command-r-plus-optimized": 3,
     "yi-large-fc": 3,
+    "deepseek-ai/DeepSeek-V2-Chat-0628": 0.14,
```
Comment (HuanzhiMao) [inline berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py:698]: This should be removed. OSS models don't have a cost.

File: berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py [hunk 2]
```diff
@@ -723,6 +758,11 @@
     "command-r-plus-FC-optimized": 15,
     "command-r-plus-optimized": 15,
     "yi-large-fc": 3,
+    "deepseek-ai/DeepSeek-V2-Chat-0628": 0.28,
```
Comment (HuanzhiMao) [inline berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py:761]: Same as above.

File: berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py (hunk 1 above)
Comment (moonlight1431) [reply berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py]: Removed costs for both parts, sorry about that, I was looking at the API costs!

**Comment by moonlight1431** [comment]
added the DeepSeek-Coder-V2 to the deepseek handler!!

**Comment by ShishirPatil** [comment]
Thank you @moonlight1431  
@HuanzhiMao  I think once we fix the merge conflict in our latest handler map, this is good to go?

**Review by HuanzhiMao — APPROVED** [review]
LGTM

**Pull request merged**